livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -302,6 +303,36 @@ async def recv_task():
 
         await asyncio.gather(send_task(), recv_task(), keepalive_task())
 
+    def _commit_all_final_events(self) -> None:
```

Comment:
What is the motivation for this behaviour, if you're closing VAD and not sending enough data before, if the user is still speaking then the END_OF_SPEECH event not being sent is expected IMO